string-manipulation-1/src/is-upper-cased.js

```diff
@@ -0,0 +1,10 @@
+/* exported isUpperCased */
+function isUpperCased(word) {
+  let answer = true;
+  for (let i = 0; i < word.length; i++) {
+    if (word[i] !== word[i].toUpperCase()) {
+      answer = false;
+    }
+  }
+  return answer;
+}
```

Comment:
No loop needed! Just compare the original `word` to the whole `word` in uppercase.
```
function isUpperCased(word) {
  return word === word.toUpperCase();
}
```